### PR TITLE
docs: add tdarr_inform to useful tools

### PR DESCRIPTION
**Pull request by deathbybandaid**

### CODE REVIEW

File: useful-tools.md [hunk 1]
```diff
@@ -327,6 +328,9 @@ There are a handful of options out there for extracting and deleting files after
 
 [Tdarr](https://tdarr.io) is a closed-source self hosted web-app for automating media library transcode/remux management and making sure your files are exactly how you need them to be in terms of codecs/streams/containers etc. Designed to work alongside [Sonarr](/sonarr)/[Radarr](/radarr) and built with the aim of modularization, parallelization and scalability, each library you add has its own transcode settings, filters and schedule. Workers can be fired up and closed down as necessary, and are split into 3 types - 'general', 'transcode' and 'health check'. Worker limits can be managed by the scheduler as well as manually.
 
+## tdarr_inform
+[tdarr_inform](https://github.com/deathbybandaid/tdarr_inform) is a custom script for Sonarr and Radarr to inform Tdarr of new/changed/deleted files without relying on filesystem events or frequent disk scanning.
```
Comment (bakerboy448) [inline useful-tools.md:332]: nit - new line after headers

Comment (deathbybandaid) [reply useful-tools.md:332]: I must be blind and not seeing the difference

Comment (Qstick) [reply useful-tools.md:332]: image

Comment (deathbybandaid) [reply useful-tools.md:332]: Ooh, I see it now!

**Pull request merged**